javatests/com/williamfiset/datastructures/linkedlist/DoublyLinkedListTest.java

```diff
@@ -305,6 +305,13 @@ public void testRandomizedIndexOf() {
     }
   }
 
+  @Test
+  public void testContains() {
+    list.add(1);
+    assertTrue(list.contains(1));
+    assertFalse(list.contains(2));
```

Comment:
I don't think you need to assert the linked list does not contain a certain value because there are an infinite number of numbers it does not contain. Perhaps check the size instead?